unstructured/partition/image.py

```diff
@@ -0,0 +1,32 @@
+from typing import List, Optional
+
+from unstructured.documents.elements import Element
+from unstructured.partition.pdf import partition_pdf_or_image
+
+
+def partition_image(
+    filename: str = "",
+    file: Optional[bytes] = None,
+    url: Optional[str] = "https://ml.unstructured.io/",
```

Comment:
Should the url here default to `/layout/image`?

Reply:
Per offline convo, we're going to do write up a design doc for the inference API and then circle back to this.